tchannel/tchannel.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+from tornado import gen
+
+from .format import Formatter
+from .formats import DEFAULT_FORMATS
+from .glossary import DEFAULT_TIMEOUT
+from .tornado import TChannel as NodeInspiredTChannel
+
+__all__ = ['TChannel']
+
+
+class TChannel(NodeInspiredTChannel):
+
+    def __init__(self, formatters=None):
+
+        # if no formatters, use default
+        if not formatters:
+            formatters = DEFAULT_FORMATS
+
+        # set formatters
+        for f in formatters:
+            # if not abc, blow up
+            if not issubclass(f, Formatter):
```

Comment:
I don't know if we need to check `isinstance`/`issubclass`. Duck typing. Just accept whatever and assume it has the required methods.